lldap_graphql/__init__.py

```
@@ -0,0 +1,204 @@
+        response.raise_for_status()
+
+        response_data: dict[str, Any] = response.json()
+        if "errors" in response_data:
+            messages: str = "; ".join(
+                err.get("message", "Unknown error") for err in response_data["errors"]
+            )
+            raise APIResponseError(f"GraphQL error: {messages}", response_data=response_data)
+
+        return response_data
+
+    def create_user(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)